APMrover2/failsafe.cpp

```diff
@@ -42,8 +42,8 @@ void Rover::failsafe_check()
         // pass RC inputs to outputs every 20ms
         last_timestamp = tnow;
         hal.rcin->clear_overrides();
-        uint8_t start_ch = 0;
-        for (uint8_t ch=start_ch; ch < 4; ch++) {
+
+        for (uint8_t ch = 0; ch < 4; ch++) {
```

Comment:
We create an useless variable as start_ch is only use in the loop, so I remove it